partcad/src/partcad/actions/shape/search.py

```
@@ -0,0 +1,43 @@
+        recursive,
+        keyword,
+        lambda project: getattr(project, "sketches").values()
+    )
+
+def search_assemblies(ctx: Context, package: str, recursive: bool, keyword: str) -> list[Assembly]:
+    return _search(
+        ctx,
+        package,
+        recursive,
+        keyword,
+        lambda project: getattr(project, "assemblies").values()
```

Ruff (0.11.9)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)